dev/com.ibm.ws.jdbc_fat_oracle/fat/src/com/ibm/ws/jdbc/fat/oracle/FATSuite.java

```diff
@@ -10,21 +10,79 @@
  *******************************************************************************/
 package com.ibm.ws.jdbc.fat.oracle;
 
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.junit.ClassRule;
 import org.junit.runner.RunWith;
 import org.junit.runners.Suite;
 import org.junit.runners.Suite.SuiteClasses;
+import org.testcontainers.containers.OracleContainer;
 
-import componenttest.custom.junit.runner.AlwaysPassesTest;
 import componenttest.topology.utils.ExternalTestServiceDockerClientStrategy;
 
 @RunWith(Suite.class)
 @SuiteClasses({
-                AlwaysPassesTest.class,
                 OracleTest.class,
                 OracleUCPTest.class
 })
 public class FATSuite {
+	
+    //TODO replace this container with the official oracle-xe container if/when it is available without a license
+    @ClassRule
+    public static OracleContainer oracle = new OracleContainer("oracleinanutshell/oracle-xe-11g");
+    
     static {
         ExternalTestServiceDockerClientStrategy.clearTestcontainersConfig();
     }
+    
+    public static void initDatabaseForOracleTest() throws SQLException {
```

Comment:
since this init method is only used in `OracleTest`, it would make more sense to have the code in that class, likewise for the UCP method/class